layers/+completion/auto-completion/funcs.el

```diff
@@ -295,7 +295,7 @@ MODE parameter must match the :modes values used in the call to
       (define-key map (kbd "C-k") 'company-select-previous)
       (define-key map (kbd "C-l") 'company-complete-selection))
     ;; Fix company-quickhelp Evil C-k
-    (lexical-let ((prev nil))
+    (let ((prev nil))
```

Comment:
You need to add `-*- lexical-binding: t; -*-` to the first line of this file as well.

Reply:
@madand indeed I should have done that, and have done it.